thermal.c

```diff
@@ -93,19 +93,19 @@ static gboolean prepare_netlink(void)
 
 	sock = nl_socket_alloc();
 	if (!sock) {
-		log(TO_ALL, LOG_ERR, "thermal: socket allocation failed.\n");
+		log(TO_ALL, LOG_WARNING, "thermal: socket allocation failed.\n");
 		return TRUE;
 	}
 
 	rc = genl_connect(sock);
 	if (rc) {
-		log(TO_ALL, LOG_ERR, "thermal: socket bind failed.\n");
+		log(TO_ALL, LOG_WARNING, "thermal: socket bind failed.\n");
```

Comment:
this one is the one that you get when there is no thermald available. imho even LOG_INFO with slightly more useful hint (something like `thermal: socket bind failed, is thermald running?`) could be useful here imho.